Aztec/Classes/Public/TextKit/TextAttachment.swift

```diff
@@ -79,6 +89,13 @@ public class TextAttachment: NSTextAttachment
     // MARK: - NSTextAttachmentContainer
 
     override public func imageForBounds(imageBounds: CGRect, textContainer: NSTextContainer?, characterIndex charIndex: Int) -> UIImage? {
+        self.textContainer = textContainer
+        switch kind {
+        case .RemoteImage:
+            startImageFetching()
+        default:
+            print("No action to do")
```

Comment:
No need to log this (as there's not much value in the log message).  Add a comment if we need to document it's ok if nothing happens.